providers/defaults.go

```diff
@@ -11,18 +11,21 @@ import "go.mondoo.com/cnquery/v11/providers-sdk/v1/plugin"
 // to tell users what providers are used for common connections, when there
 // is no other way to find out.
 var DefaultProviders Providers = map[string]*Provider{
+
 	"ansible": {
 		Provider: &plugin.Provider{
 			Name:            "ansible",
 			ID:              "go.mondoo.com/cnquery/v11/providers/ansible",
 			ConnectionTypes: []string{"ansible"},
 			Connectors: []plugin.Connector{
+
 				{
 					Name:  "ansible",
 					Use:   "ansible PATH",
 					Short: "an Ansible playbook",
 				},
 			},
+			Dependencies: []string(nil),
```

Comment:
do we need all of these initializations? by default, dependencies should be empty. 🤔 what am I missing?

Reply:
That entire file is auto generated, so it appears to just default to []string(nil) when it doesn't have any dependencies slice in the config for that provider. Having this still allows us to range over this without explicit nil checks. We can try to force it to remove this but it seems to be the default behavior when this file is being generated. 

We could add a check to only include it in the generated `defaults.go` if it actually has data but we would then also have to add a nil check before we attempt to loop through it.

Reply:
All of that said, I don't really have a preference one way or another.